src/pie_utils/sequence_tagging/encoding.py

```diff
@@ -0,0 +1,363 @@
+import logging
+from typing import List, Tuple, TypeVar
+
+from pie_utils.sequence_tagging.ill_formed import fix_encoding, remove_encoding
+
+logger = logging.getLogger(__name__)
+
+
+TypedSpan = Tuple[int, Tuple[int, int]]
+TypedStringSpan = Tuple[str, Tuple[int, int]]
+
+
+# T = TypeVar("T", str, Token)
+T = TypeVar("T")
```

Comment:
For what is this used? remove?

Reply:
I have no clue why it is there, maybe it's reference is removed when I deleted other methods from this script. So I guess we can remove it as well.